src/features/terminal/shells/bash/bashStartup.ts

```diff
@@ -71,10 +71,10 @@ async function isStartupSetup(profile: string, key: string): Promise<ShellSetupS
 
 async function setupStartup(profile: string, key: string, name: string): Promise<boolean> {
     if (shellIntegrationForActiveTerminal(name, profile)) {
+        removeStartup(profile, key);
```

Comment:
No zsh?

Reply:
Yeah, it seems like right now both bash and zsh share the same file..https://github.com/microsoft/vscode-python-environments/blob/66ff5dd97bffa1e3ff7d4a5e81740eb7f36296bf/src/features/terminal/shells/bash/bashStartup.ts#L187 

Might be good item for me to go refactor during debt week